recipe/run_test.sh

```diff
@@ -23,6 +23,10 @@ h5c++ -showconfig
 h5c++ ${CXXFLAGS} ${LDFLAGS} h5tutr_cmprss.cpp -o h5tutr_cmprss
 ./h5tutr_cmprss
 
+echo "test for hdf5 C++ exceptions"
+h5c++ ${CXXFLAGS} ${LDFLAGS} testhdf5exc.cpp -o testhdf5exc
```

Comment:
@gillins @xylar you can't use `${CXX}` with HDF5 you should be using h5c++ to capture all the flags needed (i think....)

Reply:
I think this test can be removed, it didn't help prove anything...

Reply:
ok not doing any harm so i'll leave it for now, but I might just rebase things later.

Gotta get some time to look into the osx-arm failure.